test/pytest/test_handler.py

```diff
@@ -224,6 +224,27 @@ def test_kfserving_mnist_model_register_and_inference_on_valid_model_explain():
 
     assert np.array(json.loads(response.content)['explanations']).shape == (1, 1, 28, 28)
     test_utils.unregister_model("mnist")
+
+def test_mnist_batch_inference():
```

Comment:
where is this test case called? And also what's the difference b/s test_mnist_batch_inference vs testSuccessBatch?

Reply:
Purely a frontend vs backend test - test is called if you run pytest here